domain/domain.go

```diff
@@ -358,10 +363,31 @@ func (do *Domain) Reload() error {
 
 	// Lock here for only once at the same time.
 	do.m.Lock()
-	defer do.m.Unlock()
+	defer func() {
+		do.lastReloadAt = time.Now()
+		do.m.Unlock()
+	}()
 
 	startTime := time.Now()
-
+	if do.lastReloadAt != emptyTime {
+		reloadInterval := startTime.Sub(do.lastReloadAt)
+		intervalOffset := reloadInterval - do.ddlLease/2
+		if intervalOffset >= 0 {
+			metrics.LoadSchemaIntervalBias.Observe(float64(intervalOffset.Milliseconds()))
+		}
+		if intervalOffset >= do.ddlLease {
+			// more than 2 * ddlLease
+			metrics.LoadSchemaIntervalWarning.WithLabelValues("one_ddl_lease").Inc()
+		} else if intervalOffset > do.ddlLease/2 {
+			// a little more than half ddlLease
```

Comment:
I am a bit confusing, the first branch means `>= 3/2 * ddlLease`, the 2nd branch means `> ddlLease`. But the comment is not the case?

Reply:
first branch means greater than a lease, second branch means less than a lease and greater than half lease.

Reply:
OK, I am not that familiar. So, `intervalOffset := reloadInterval - do.ddlLease/2` is needed and intended, right?

Reply:
Yes